modules/internal/Atomics.chpl

```diff
@@ -413,6 +413,11 @@ module Atomics {
       x.write(read());
     }
 
+    pragma "no doc"
+    proc const encodeTo(x) throws {
```

Comment:
I'll look through these and see which ones can be removed. I know in the array case we still need an explicit ``encodeTo`` to avoid some problems with triggering promotion in the compiler-generated case. Something to do with the last-resort pragma, I presume.

Reply:
You're welcome to try to remove some but I also think it's fine to leave them. I just wanted to make sure I understood the situation.